### PR TITLE
[SYNPY-1384] Create uploading data in bulk tutorial

### PR DESCRIPTION
**Pull request by BryanFauble**
**Problem:**

1. We had no uploading data in bulk tutorial
2. Some of the tutorials used a directory structure that did not match the flattened layout
3. Some install instructions were confusing and led to some folks having issues as they ran what we had provided

**Solution:**

1. Creating the uploading data in bulk tutorial
2. Revamp some tutorials to follow the flattened data layout
3. Updated install commands
4. Removed some copy/pasted scripts. Instead dynamically showing specific lines of python in the tutorial

**Testing:**

1. I stepped through all of the tutorials in order and verified that I was able to run them with the expected outcome.
2. Verified I could run the install commands as provided.

### CODE REVIEW

File: docs/tutorials/python/tutorial_scripts/upload_data_in_bulk.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+PATH_TO_MANIFEST_FILE = os.path.expanduser(os.path.join("~", "manifest-for-upload.tsv"))
+
+# Step 1: Let's find the synapse ID of our project:
+my_project_id = syn.findEntityId(
+    name="My uniquely named project about Alzheimer's Disease"
+)
+
+# Step 2: Create a manifest TSV file to upload data in bulk
+# Note: When this command is run it will re-create your directory structure within
+# Synapse. Be aware of this when running this command that this will occur.
```
Comment (BWMac) [inline docs/tutorials/python/tutorial_scripts/upload_data_in_bulk.py:23]:
```suggestion
# Synapse. Be aware of this before running this command.
```

**Review by BWMac — APPROVED** [review]
LGTM! I followed along and went through the updated docs and everything looks good and worked as expected.

**Review by thomasyu888 — COMMENTED** [review]
🔥 Thanks for the review and work! Just tiny comment.

File: docs/tutorials/python/annotation.md [hunk 2]
```diff
@@ -19,58 +25,19 @@ In this tutorial you will:
...
-)
-
-# Retrieve the folders I want to annotate files in
-batch_1_folder_id = syn.findEntityId(
-    name="single_cell_RNAseq_batch_1", parent=my_project_id
-)
-
-print(f"Batch 1 Folder ID: {batch_1_folder_id}")
-
+{!docs/tutorials/python/tutorial_scripts/annotation.py!lines=5-22}
```
[the remark this answers is not in the document]
Comment (BryanFauble) [reply docs/tutorials/python/annotation.md:28]: I agree there is some risk here. My opinion is its a risk by manually copying the code over. At least in this way the code gets updated and the lines for each step is updated. I guess it'll be something to look out for in future PRs if there are changes.

**Pull request merged**